frontend/src/modules/member/member-routes.js

```diff
@@ -35,19 +33,6 @@ export default [
           PermissionGuard(LfPermission.memberRead),
         ],
       },
-      {
-        name: 'memberEdit',
```

Comment:
This is still being called in `contributor-dropdown.vue`, `member-actions.vue`, `member-dropdown-content.vue`. So on these component, the actions will break because they expect this page to be there.
I think most of them say "Edit profile":
1. If the option is inside the profile page, remove this option
2. If the option is outside the profile page, redirect to the profile page

Also the dropdown in the profile style is a bit broken because we are still showing the divider
![Screenshot 2024-07-26 at 17 20 47](https://github.com/user-attachments/assets/2424a7eb-22b5-4011-9183-e0bcc7d68640)